src/pages/drive.tsx

```diff
@@ -86,8 +86,6 @@ const Drive = ({
       <Layout title={metatags[0].title} description={metatags[0].description} segmentName="Drive" lang={lang}>
         <Navbar textContent={navbarLang} lang={lang} cta={['default']} fixed />
 
-        <ProductsNavigation textContent={navbarLang} lang={lang} selectedItem="drive" />
```

Comment:
If this component is not longer used, remove it too.

Reply:
Checked that is not used in other page and removed